### PR TITLE
gedit 3.18.2

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

**Comment by xu-cheng** [comment]
I finally see why it requires brew python to work.

**Comment by mikemcquaid** [comment]
CC @tschoonj https://github.com/Homebrew/homebrew/pull/45932

**Comment by tschoonj** [comment]
Ok looks good to me.

Thanks for the work @xu-cheng

File: Library/Formula/gobject-introspection.rb [hunk 1]
```diff
@@ -17,7 +17,10 @@ class GobjectIntrospection < Formula
   depends_on "glib"
   depends_on "cairo"
   depends_on "libffi"
-  depends_on :python if MacOS.version <= :mavericks
+  # System python in Mavericks or below has bug in distutils/sysconfig.py, which breaks the install.
+  #    Caught exception: <type 'exceptions.AttributeError'> AttributeError("'NoneType' object has no attribute 'get'",)
+  #    > /System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/sysconfig.py(171)customize_compiler()
+  depends_on "python" if MacOS.version <= :mavericks
```
Comment (mikemcquaid) [inline Library/Formula/gobject-introspection.rb]: :clap:

**Pull request closed without merging**